extra_data/export.py

```diff
@@ -242,7 +255,31 @@ def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
     streamer.start()
     for tid, train_data in data.trains():
         if train_data:
-            streamer.feed(train_data)
+            if append_detector_modules:
+                if source_glob == '*':
+                    warn(' You are trying to stack detector-module sources \
+with a global windcard (\'*\'). If there are non-detector sources in your \
+run, this will fail.\n')
```

Comment:
```suggestion
                    warn("You are trying to stack detector-module sources "
                         "with a global windcard ('*'). If there are non-detector "
                         "sources in your run, this will fail.")
```

Reply:
Same here, thanks